src/components/List/Grid.js

```diff
@@ -1,21 +1,17 @@
-import React from 'react'
-import './Grid.styles.css'
+import React from 'react';
+import './Grid.styles.css';
 
 import Single from './Single';
 
 function Grid({ items }) {
-    const renderList = () => {
-        return items.map((item, index) => (
-          <Single key={index} item={item}/>
-        ));
-      }
-    return (
-        <div className='row'>
-            <ul>
-                {renderList()}
-            </ul>
-        </div>
-    )
+  const renderList = () => {
+    return items.map((item, index) => <Single key={index} item={item} />);
+  };
+  return (
+    <div className="row">
+      <ul>{renderList()}</ul>
+    </div>
+  );
```

Comment:
Aunque llamar una función dentro de un componente no es malo, hay formas más correctas para hacerlo, aquí te va un posible refactor:
```suggestion
  const VideosList = () => {
    return (
      <ul>
        {items.map(
          (item, index) => <Single key={index} item={item} />
        )}
      </ul>
    );
  };

  return (
    <div className="row">
      <VideosList />
    </div>
  );
```